docs/usage/topology_aware_routing.md

```diff
@@ -0,0 +1,96 @@
+# Topology-Aware Traffic Routing
+
+The `TopologyAwareRouting` feature enables topology-aware traffic routing for components in the garden or seed clusters.
+
+## Motivation
+
+The enablement of [highly available Shoot control planes](shoot_high_availability.md) requires multi-zone Seed clusters. A Garden cluster can also be a multi-zone cluster. The topology-aware routing is introduced to reduce costs and to improve network performance for the cross availability zone traffic. The cross availability zone traffic is charged by the cloud providers and it comes with higher latency compared to the traffic within the same zone. The `TopologyAwareRouting` feature enables topology-aware routing for Services deployed in the garden or seed clusters. For the clients consuming these topology-aware Services, kube-proxy prefers the endpoints which are within the same zone.
```

Comment:
I am a bit puzzled by the sentence `The topology-aware routing is introduced to reduce costs and to improve network performance for the cross availability zone traffic.`
Topology-aware routing may reduce costs and improve network performance by omitting/preventing cross-zonal traffic if possible.
I suppose you mean the same thing, but I had trouble understanding it like that.

Reply:
Valid point. The wording should be improved with the latest commit.